docs/topics/proc_exposing-application-metrics-using-prometheus-with-vertx.adoc

```diff
@@ -0,0 +1,189 @@
+
+[id='exposing-application-metrics-using-prometheus-with-vertx_{context}']
+= Exposing application metrics using Prometheus with {Vertx}
+
+In this example, you:
+
+* Configure your application to expose metrics.
+* Collect and view the data using Prometheus.
+
+Prometheus connects to a monitored application to collect data; the application does not send metrics to a server.
+
+.Prerequisites
+
+* Prometheus configured to collect metrics from the application:
++
+--
+. Download and extract the link:https://prometheus.io/download/[archive^] with the latest Prometheus release:
++
+[source,bash,options="nowrap"]
+----
+$ wget https://github.com/prometheus/prometheus/releases/download/v2.4.3/prometheus-2.4.3.linux-amd64.tar.gz
+$ tar -xvf  prometheus-2.4.3.linux-amd64.tar.gz
+----
+
+. Navigate to the directory with Prometheus:
++
+[source,bash,options="nowrap"]
+----
+$ cd  prometheus-2.4.3.linux-amd64
+----
+
+. Append the following snippet to the `prometheus.yml` file to make Prometheus automatically collect metrics from your application:
++
+[source,yaml,options="nowrap"]
+----
+  scrape_configs:
+    - job_name: 'prometheus'
+      static_configs:
+      - targets: ['localhost:9090']
+    - job_name: 'vertx-app'
+      static_configs:
+      - targets: ['localhost:8081']
+----
++
+The default behavior of {Vertx}-based applications is to expose metrics at the `/metrics` endpoint.
+This is what the MicroProfile Metrics specification requires, and also what Prometheus expects.
```

Comment:
I'm not sure the MicroProfile Metrics specification reference is relevant here.

Reply:
It is not.

Reply:
Good catch. Thanks.